modules/acknowledgements/js/filter.js

```diff
@@ -0,0 +1,98 @@
+/* global fetchAll */
+$(document).ready(
```

Comment:
If this is supposed to filter the table, you have a `SelectionFilter` react component for that